### ISSUE TITLE
ContainerExceptionInterface is not throwable

### ISSUE THREAD

**Issue by hultberg** [php-fig/container#21]
Hello. I have been following PR #20 and I see that the exception interfaces will [extend `Throwable` in it](https://github.com/php-fig/container/pull/20/files#diff-78f086142adda0db42e1deecaa6a56c6R11), along with the breaking changes with type-hints. However, the static code analyser phpstan is now reporting that the exception interface is not a subtype of Throwable as its an error:
```
PHPDoc tag @throws with type Psr\Container\ContainerExceptionInterface is not subtype of Throwable
```

Is there a possibility to fix this outside of PR #20 ? I'm not familiar with any bylaws of FIG and what the possibilities are.

**Comment by Jean85** [comment]
We've been discussing this kind of issues a lot lately, and we've condensed our proposed solution to updating PSR interfaces in a blog post. Please read it and provide your feedback!

https://www.php-fig.org/blog/2019/10/upgrading-psr-interfaces/

**Comment by stof** [comment]
@weierophinney should be closed as 2.0.0 fixes this (1.1.0 as well but reverted in 1.1.1 due to the BC break)
